doc/getting-started/getting-started.rst

```diff
@@ -205,10 +186,30 @@ hardware.
 
 .. important::
 
-   Whenever you change the configuration of the board, such as peripherals, BIOS
-   settings, additional memory, or PCI devices, you must generate a new board
+   Before running the Board Inspector, you must set up your target hardware and
+   BIOS exactly as you want it, including connecting all peripherals,
+   configuring BIOS settings, and adding memory and PCI devices. For example,
+   you must connect all USB devices; otherwise, the Board Inspector will not
```

Comment:
Does that mean the Board Inspector does not see the USB ports if nothing is plugged in? If true, that sounds like something we need to fix!

Reply:
@NanlinXie @dbkinder Does the wording accurately describe the USB issue that a customer encountered (USB ports?)? Also, should we create a JIRA ticket?

Reply:
OK, I see that too on my testbed. I do not feel this is the correct behaviour however, as USB ports are given to a VM and it does (shouldn't) matter if there is any device plugged into the port (and what device it is). The correct behaviour in my opinion should be that **all** ports should show up in the drop-down menu (and the device plugged in could be indicated if there is any).

Reply:
@gvancuts Create a ticket?